cmd/eksctl-anywhere/cmd/flags.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+func addClusterFlags(cc *cobra.Command, clusterOpt *clusterOptions, override ...string) {
```

Comment:
The fact there's an override (that should be called 'exclude') tells me this isn't common code. I'd rather see individual functions such as `applyBundlesOverrideFlag(...)` called in the various commands its required because its abundantly clear whats happening and makes that flag consistent across commands.

Reply:
Thanks Chris, I removed the override arg and reduced addClusterFlags to only flags used on all 3 (create, upgrade, delete) commands. The `delete cluster` command does have a different usage statement for filename so I added an overrideFlagNameAndUsage in flags.go.

Let me know if that looks like a better way of handling the common flags, or if it may be better to just keep the flag initialization separate per command.

Reply:
Decided to back out of the flag grouping changes. If needed, they can be handled in a separate PR